### PR TITLE
fix: to correct response status code

### PR DESCRIPTION
**Pull request by orisano**
I encountered a problem that nginx response status code is 000.
Maybe this problem happened since #62.

I fixed to respond always 200 OK.
question: Is it right to always return 200?

### CODE REVIEW

**Review by kumagi — CHANGES_REQUESTED** [review]
AFAIK, the status code of a SXG file should be always 200, no other 20* response is expected.
Is it true? @twifkak

File: .github/workflows/end-to-end-test.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@ jobs:
         run: |
           if grep -qi "etag" test/out/index.header ; then exit 1; fi
 
+      - name: check status code is must be 200
```
[the remark this answers is not in the document]
Comment (orisano) [reply .github/workflows/end-to-end-test.yml]: I fixed it

File: .github/workflows/end-to-end-test.yml [hunk 2]
```diff
@@ -49,6 +49,10 @@ jobs:
         run: |
           if grep -qi "etag" test/out/index.header ; then exit 1; fi
 
+      - name: check status code to be 200
```
Comment (kumagi) [inline .github/workflows/end-to-end-test.yml:52]: Capitalize please.

**Review by kumagi — CHANGES_REQUESTED** [review]

**Comment by kumagi** [comment]
LGTM, thank you @orisano , I'll wait for respones of @twifkak until next working day.

**Review by kumagi — APPROVED** [review]

**Comment by orisano** [comment]
what is status of this pr now?

**Comment by twifkak** [comment]
Sorry for the delay; extended vacation. These changes seem good to me. Thank you for the test! I will merge.

I don't believe an "outer" response status of 200 is **required**, per the [loading spec](https://wicg.github.io/webpackage/loading.html#identifying-sxg), but I may be wrong. I would need to test to be certain.

However, 200 seems like the semantically correct choice (and certainly 000 is wrong).

**Comment by orisano** [comment]
thanks! we are waiting for the next release.

**Pull request merged**